client/src/components/CreateCard/cardContext.js

```diff
@@ -80,13 +80,13 @@ const CardProvider = props => {
           description,
           tag
         };
-        authFetch(
-          `dashboards/${dashboardId}/columns/${columnId}/tasks/${task}`,
-          {
-            method: "PUT",
-            body: JSON.stringify(updatedTask)
-          }
-        )
```

Comment:
please return this file to original, also please try to make it a habit to remove unecessary console.log before pr

Reply:
Thank you for reporting.
I returned it to original